backend/fbdev.c

```diff
@@ -88,6 +123,27 @@ static bool twin_fbdev_work(void *closure)
     return true;
 }
 
+static bool twin_fbdev_is_rgb565(twin_fbdev_t *tx)
```

Comment:
Use `static inline` instead to aid compiler optimizations. That is,
```c
static bool twin_fbdev_is_rgb565(const twin_fbdev_t *tx)
...
```

Same for `twin_fbdev_is_rgb888` and `twin_fbdev_is_argb32`.